pkg/api/mlflow/service/run/service.go

```diff
@@ -520,7 +520,11 @@ func (s Service) LogParam(
 
 	param := convertors.ConvertLogParamRequestToDBModel(run.ID, req)
 	if err := s.paramRepository.CreateBatch(ctx, 1, []models.Param{*param}); err != nil {
-		return api.NewInternalError("unable to insert params for run '%s': %s", run.ID, err)
+		if _, ok := err.(repositories.ParamConflictError); ok {
```

Comment:
probably we can just use `errors.Is` in that case?

Reply:
errors.As is working, changed to that